NEXT_CHANGELOG.md

```diff
@@ -27,6 +27,24 @@ Description! And a link to a [reference](http://url)
 ## ❗ BREAKING ❗
 
 ## 🚀 Features
+### Add more customizable metrics ([PR #1159](https://github.com/apollographql/router/pull/1159))
+Added the ability to add custom attributes/labels on metrics via the configuration file.
+Example:
+```yaml
+telemetry:
+  metrics:
+    common:
+      additionnal_attributes:
```

Comment:
Typo `additionnal` -> `additional`. This also needs to be fixed in the code.

Reply:
Maybe even just get rid of the `additional_` prefix altogether.

```yaml
telemetry:
  metrics:
    common:
      attributes:
      ...
```      

This way in future we can add more config to the attributes section, such as `default: false`, to disable the default attributes.